config/examples/core.yml

```diff
@@ -1,6 +1,6 @@
 base: &default
   url: <%= ENV.fetch('THREESCALE_CORE_INTERNAL_API', 'http://localhost:3001/internal/') %>
-  fake_server: 'http://localhost:3000/internal/'
+  fake_server: <%= ENV.fetch('THREESCALE_FAKE_SERVER', 'http://localhost:3000/internal/') %>
```

Comment:
The `THREESCALE_FAKE_SERVER` env var was defined in `config/docker/core.yml`, and I though it would be nice to make this configurable in the example file.

Reply:
Personally I think we don't need this variable locally. Not a blocker but just one more variable nobody would use.